ReactAndroid/src/main/java/com/facebook/react/ReactActivityDelegate.java

```diff
@@ -15,8 +15,6 @@
 
 import com.facebook.infer.annotation.Assertions;
 import com.facebook.react.bridge.Callback;
```

Comment:
Before looking to merge this - does it make sense to have a `ReactActivityDelegate` and then a `ReactDelegate` or does it make sense for `ReactActivity` to use the `ReactDelegate` directly? Was trying to bring smaller changes with this PR and focus primarily on the addition of the `ReactFragment` but also don't want to incur possible Tech Debt